src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -133,4 +135,15 @@ public String removeDatasetAssetSpecifications(
             .addParameter(JobMapKeys.ASSET_ID.getKeyName(), assetId)
             .submit();
     }
+
+    public String deleteDatasetData(
```

Comment:
Similar question as on the API, should this name be more specific since it's only dealing with tables?